en/docs/develop/creating-artifacts/creating-an-api.md

```diff
@@ -1,73 +1,71 @@
+              <div class="admonition note">
+              <p class="admonition-title">Note</p>
+              <p>
+              <ul>
+                 <li>It is not recommended to use the same API context in multiple APIs. The API context should be unique for each API.</li>
+                 <li>It is also not recommended to use <code>/service</code> as API context because it is preserved for Proxy services.</li>
+              </ul>
+              </p>
+              </div>
+              </td>
+           </tr>
+           <tr>
+         <td>Generate API From</td>
+         <td>
+             Select one of the given options:
              <ul>
-                <li>It is not recommended to use the same API context in multiple APIs. The API context should be unique for each API.</li>
-                <li>It is also not recommended to use <code>/service</code> as API context because it is preserved for Proxy services.</li>
+                 <li>None: This option is selected by default. Use this option if you want to create a new REST API artifact from scratch.</li>
+                 <li>From OpenAPI definition: Select this option if you want to generate the REST API artifact from an existing Swagger definition (YAML/JSON file). That is, the synapse configuration (XML) of the REST API will be generated using the Swagger definition.</li>
+                 <li>From WSDL file: Select this option to generate Synapse API from a WSDL endpoint.</li>
              </ul>
-             </p>
-             </div>
-             </td>
-          </tr>
-          <tr>
-             <td>Version Type</td>
-             <td></td>
-          </tr>
-          <tr>
-        <td>Generate API From</td>
-        <td>
-            Select one of the given options:
-            <ul>
-                <li>None: This option is selected by default. Use this option if you want to create a new REST API artifact from scratch.</li>
-                <li>From OpenAPI definition: Select this option if you want to generate the REST API artifact from an existing Swagger definition (YAML/JSON file). That is, the synapse configuration (XML) of the REST API will be generated using the Swagger definition.</li>
-                <li>From WSDL file: Select this option to generate Synapse API from a WSDL endpoint.</li>
-            </ul>
-        </td>
-        </tr>
-       </tbody>
-    </table>
+         </td>
+         </tr>
+        </tbody>
+     </table>
 
     <a href="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/create-new-rest-api.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/create-new-rest-api.png" alt="Create new rest API" width="80%"></a>
 
     <br/>
 
     - If you selected **None** in the previous step, you can start creating an API from scratch.
-    
+
     -  If you selected **From OpenAPI definition** in the previous step, enter the details of your custom Swagger file:
-    
+
```

Comment:
Click <b>Select Location</b> and select a file for OpenAPI definition. This is a required field. -> 

 Click <b>Select Location</b> and select a file for OpenAPI definition. This field is required.